release-notes/opensearch-alerting-dashboards-plugin.release-notes-3.0.0.0-alpha1.md

```diff
@@ -0,0 +1,10 @@
+## Version 3.0.0.0-alpha1 Release Notes
+
+Compatible with OpenSearch 3.0.0.0-alpha1
+
+### Maintenance
+* [Release 3.0] Add alpha1 qualifier. ([#1218](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/1218))
```

Comment:
change this to 1224?

Reply:
Discussed offline. No need to change this reference.